.eslintrc.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   'extends': 'eslint-config-brigade/react',
-  'ecmaFeatures': {
-    'modules': true,
+  'parserOptions': {
+    'ecmaVersion': 6,
+    'sourceType': 'module'
```

Comment:
That´s correct!
Is for migrate to eslint 2: http://eslint.org/docs/user-guide/migrating-to-2.0.0#language-options